src/pages/working/index.tsx

```diff
@@ -25,6 +26,7 @@ import {
 } from '@/components';
 
 import './Working.scss';
+import { Sidebar } from '../../components/sidebar';
```

Comment:
korzystuj z prefixu ---> import ... from "@/components"
Jeśli stworzyłeś import wyżej w pliku do reeksportowania plików to korzystaj z niego, w tym przypadku import wyglądałby tak:
import { ..., ...., ..., Sidebar }  from "@/components"
Dodatkowo z tego co widzę do importy nie są posortowanie, eslint Ci nie krzyczał na wszystkie importy?